README.md

```diff
@@ -15,91 +15,69 @@ and using:
 )
 
 func main() {
-	svc, err := urlshortener.New(http.DefaultClient)
-	// ...
+        ctx := context.Backgroud()
+        svc, err := urlshortener.NewService(ctx)
+        // ...
 }
 ```
 
 * For a longer tutorial, see the [Getting Started guide](https://github.com/google/google-api-go-client/blob/master/GettingStarted.md).
 * For examples, see the [examples directory](https://github.com/google/google-api-go-client/tree/master/examples).
 * For support, use the [golang-nuts](https://groups.google.com/group/golang-nuts) mailing list.
-* The code review instance may be found [here](https://code-review.googlesource.com).
 
 ## Status
-[![Go Reference](https://pkg.go.dev/badge/google.golang.org/api.svg)](https://pkg.go.dev/google.golang.org/api)
-
-These are auto-generated Go libraries from the Google Discovery Service's JSON description files of the available "new style" Google APIs.
 
-Due to the auto-generated nature of this collection of libraries, complete APIs or specific versions can appear or go away without notice.
-As a result, you should always locally vendor any API(s) that your code relies upon.
+[![Go Reference](https://pkg.go.dev/badge/google.golang.org/api.svg)](https://pkg.go.dev/google.golang.org/api)
 
-These client libraries are officially supported by Google.  However, the libraries are considered complete and are in maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
+These are auto-generated Go libraries from the Google Discovery Service's JSON description files.
 
-If you're working with Google Cloud Platform APIs such as Datastore or Pub/Sub,
-consider using the
-[Cloud Client Libraries for Go](https://github.com/googleapis/google-cloud-go)
-instead. These are the new and
-idiomatic Go libraries targeted specifically at Google Cloud Platform Services.
+Due to the auto-generated nature of this collection of libraries they may contain breaking changes from one release to
+the next.
 
-The generator itself and the code it produces are beta. Some APIs are
-alpha/beta, and indicated as such in the import path (e.g.,
-"google.golang.org/api/someapi/v1alpha").
+These client libraries are officially supported by Google.  However, the libraries are considered complete and are in
+maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
 
-## Application Default Credentials Example
+If you're working with Google Cloud Platform APIs such as Datastore or Pub/Sub, please use the
+[Cloud Client Libraries for Go](https://github.com/googleapis/google-cloud-go) instead. These are the new and idiomatic
+Go libraries targeted specifically at Google Cloud Platform Services.
 
-Application Default Credentials provide a simplified way to obtain credentials
-for authenticating with Google APIs.
+The generator itself and the code it produces are beta. Some clients are in alpha/beta and indicated as such in the
```

Comment:
The code is beta but the client is alpha/beta?

Reply:
Removed second line and moved this up to the section on stability.